app/views/DR/HomeScreen/index.js

```diff
@@ -112,7 +118,31 @@ class HomeScreen extends Component {
     this.setState(state => ({ ...state, refreshing: true }), this.getCases);
   };
 
-  componentDidMount() {
+  handlerPositiveState = () => {
+    this.setState({ notified: false });
+    this.props.navigation.navigate('PositiveOnboarding', {
+      positive: true,
+      use: this.state.useType,
+    });
+  };
+
+  async componentDidMount() {
+    const covidIdList = JSON.parse(await GetStoreData('covidIdList'));
+    const haveBeenNotified = await GetStoreData('haveBeenNotified');
+
+    if (covidIdList !== null && !haveBeenNotified) {
+      const promiseList = covidIdList.map(userState => {
+        return fetch(`${FIREBASE_SERVICE}/covid-state/${userState.covidId}`);
+      });
+
+      Promise.all(promiseList).then(state => {
+        const checkState = state.find(({ data }) => data.positive === true);
+        if (checkState) {
+          SetStoreData('haveBeenNotified', true);
+          this.handler(true, 'mySelf');
```

Comment:
The state will be set with the same data if more than two are positive.

Could we just set the states once if at least one if positive?

Reply:
in this part its looking for at least one of the covidId to be positive

Reply:
Gotya!